base/applications/mscutils/eventvwr/eventvwr.c

```diff
@@ -511,7 +511,7 @@ BOOL
 SaveSettings(VOID)
 {
     HKEY hKeyEventVwr;
-    DWORD dwSize;
+    SIZE_T dwSize;
```

Comment:
RegSetValueExW takes DWORD as its last parameter. So in x64 I wonder how this will be handled: the (larger) SIZE_T will be truncated to a DWORD?

Reply:
I added a cast to wcslen() isntead, which is safe on a static buffer.